client/src/pages/Profile/functions.jsx

```diff
@@ -1,19 +1,45 @@
-export function getTasks() {
-  return [
-    {
-      name: 'Welcome to your F!rosh profile page',
-      description:
-        "Take a look around and explore! You'll find your personalized Frosh group schedule and more information here. You can check this off when you're ready!",
-      dateCreated: new Date('2022-05-31T00:00:00'),
-      completed: true,
-    },
-    {
-      name: 'Welcome to your F!rosh profile page 2',
-      description: 'More talky talk and walky walk text...',
-      dateCreated: new Date('2022-05-31T00:00:00'),
-      completed: true,
-    },
-  ];
+import useAxios from '../../hooks/useAxios';
+
+const { axios } = useAxios();
+
+// function checks if email is valid and sends a reset password email
+export async function resetPassword(email) {
+  console.log(email);
+  let promise = new Promise((res, rej) => {
+    setTimeout(() => res(''), 1000);
+    // currently does not display this ^ response as the display for error message,
+
+    // uses the result to check what response to display
+    // false -> "We didn't recognize that email, please try again!"
+    // true  -> "Success, an email has been sent!"
+  });
+  let result = await promise;
+  return true;
+}
+
+export async function getTasks() {
+  try {
+    const response = await axios.get('/announcements');
+    console.log(response);
+    return response.data.announcements;
+  } catch (error) {
+    console.log(error);
+  }
+  // return [
+  //   {
+  //     name: 'Welcome to your F!rosh profile page',
+  //     description:
+  //       "Take a look around and explore! You'll find your personalized Frosh group schedule and more information here. You can check this off when you're ready!",
+  //     dateCreated: new Date('2022-05-31T00:00:00'),
+  //     completed: true,
+  //   },
+  //   {
+  //     name: 'Welcome to your F!rosh profile page 2',
+  //     description: 'More talky talk and walky walk text...',
+  //     dateCreated: new Date('2022-05-31T00:00:00'),
+  //     completed: true,
+  //   },
+  // ];
```

Comment:
The current announcement model schema only has a description, but since the get endpoint for /announcements returns an array of all the announcements, no changes need to be made to this function when the announcement model is fixed.